### DISCUSSION TITLE
quarto-pub API error 413

### DISCUSSION THREAD

**Discussion by ashleyleech** [quarto-dev/quarto-cli#3800] [Q&A]
Hello, 

When publishing my website via "quarto publish quarto-pub", I am getting the following error: "ERROR: API Error: 413 (The maximum publish size is 100MB)." I am not able to find a way to fix this within Quarto. Any suggestions? 

Thank you for any help you may be able to provide!

**Comment by mcanouil** [comment]
I believe this is a Quarto Pub(web hosting service) limitation and not a Quarto CLI issue.
Since it is free, it's unlikely you'll get unlimited storage.
100 Mb should be enough for most documents.
For bigger website, you should consider other hosts.

**Reply by ashleyleech** [reply]
Thank you!
